integration-tests/reactive-messaging-kafka/pom.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-reactive-messaging-kafka</artifactId>
```

Comment:
We can also add name and description following this pattern:

https://github.com/quarkusio/quarkus/blob/efc6dc8e43599d3d574c468fdf70a02e1f226acb/integration-tests/reactive-messaging-amqp/pom.xml#L14-L15

Reply:
Added to the latest commit.